javagen/src/main/java/com/azure/autorest/model/clientmodel/ClientMethod.java

```diff
@@ -397,15 +397,23 @@ public void addImportsTo(Set<String> imports, boolean includeImplementationImpor
                     imports.add("com.azure.core.util.serializer.TypeReference");
                     imports.add("java.time.Duration");
 
-                    if (getMethodPollingDetails().getPollingStrategy() != null) {
+                    if (getMethodPollingDetails() != null) {
                         List<String> knownPollingStrategies = Arrays.asList(
```

Comment:
Could this be turned into a static field?

Reply:
Done.